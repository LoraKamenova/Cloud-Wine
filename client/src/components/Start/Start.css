.start-page {
    background-image: url('https://res.cloudinary.com/dyhmxus4n/image/upload/v1615655024/React%20Project/home_xcupse.jpg');
    background-position: center center;
    background-repeat: no-repeat;
    background-attachment: fixed;
    background-size: 100% 100%;
    height: 100vh;
    display: flex;
    flex-direction: column;
}

.start-main-container {
    display: flex;
    flex-direction: column;
    justify-content: space-between;
    flex-grow: 1
}
