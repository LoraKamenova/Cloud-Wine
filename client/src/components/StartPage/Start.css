.start-section {
    display: flex;
    flex-direction: column;
    align-self: flex-start;
}

.header-container {
    height: 30%;
}
