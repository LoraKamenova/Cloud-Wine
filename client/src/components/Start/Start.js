import './Start.css'
import Header from "../Header/Header";

const Start = () => {
    return (
        <section className="custom-section start-section">
            <Header/>
        </section>
    );
};

export default Start;
