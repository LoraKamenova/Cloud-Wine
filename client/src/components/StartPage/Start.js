import './Start.css'
import Header from "../Navigation/Header/Header";

import {Component} from 'react';

class Start extends Component {
    constructor(props) {
        super(props)
    }

    render() {
        return (
            <section className="start-section">
                <div className="header-container">
                    <Header/>
                </div>
            </section>
        )
    }
}

export default Start;
