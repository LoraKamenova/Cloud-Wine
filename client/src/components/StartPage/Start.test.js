import { render } from '@testing-library/react';
import Start from './Start';

describe('Start Component', () => {

    it("Start Component snapshot", () => {
        const { asFragment } = render(<Start />);
        expect(asFragment()).toMatchSnapshot();
    });

});
