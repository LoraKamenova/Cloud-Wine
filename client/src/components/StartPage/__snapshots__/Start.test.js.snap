// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Start Component Start Component snapshot 1`] = `
<DocumentFragment>
  <section
    class="start-section"
  >
    <div
      class="header-container"
    >
      <p
        class="header-heading"
      >
        Винописи
      </p>
    </div>
  </section>
</DocumentFragment>
`;
